README.md

```diff
@@ -1 +1,40 @@
 # neon-examples
+
+![Github Actions](https://github.com/neon-bindings/examples/workflows/Test/badge.svg?branch=main)
+
+Examples and common patterns for [Neon][neon].
+
+All examples are for [`napi-backend`][napi-migration]. For examples using `legacy-backend` see the [`legacy`][legacy] branch. 
+
+[neon]: https://github.com/neon-bindings/neon
+[napi-migration]: https://github.com/neon-bindings/neon/blob/main/MIGRATION_GUIDE.md#n-api-migration-guide
+[legacy]: https://github.com/neon-bindings/examples/tree/legacy
+
+## Table of Contents
+
+| Example                      | Description                                |
+| ---------------------------- | ------------------------------------------ |
+| [`hello-world`][hello-world] | Return a JS String with a greeting message |
+
+[hello-world]: examples/hello-world
+
+## Contributing
+
+### Setup
+
+[`pnpm`](https://www.npmjs.com/package/pnpm) provides monorepo workspace tooling. 
```

Comment:
Is it worth mentioning that Neon doesn't depend on pnpm, it's just what we're using for this repo?

Reply:
NPM now has native workspaces support:
https://docs.npmjs.com/cli/v7/using-npm/workspaces

Reply:
Unfortunately, it doesn't provide the functionality we need. The workspace features built into `npm` are very minimal. They provide linking and that's about it.

We at least need recursive building, but also benefit from module deduplication.

Reply:
I see. Which examples require being built recursively?

Reply:
Not really recursively, but the `examples/*` glob. The `npm` workspaces feature won't run `npm install` on all of them.

I really hope I'm wrong. 😁  I would love to remove `pnpm` and make the examples repo a little easier to understand, if possible.

Reply:
We can work around that by following the existing structure and using explicit paths:

```json
  "workspaces": [
    "a",
    "b",
    ...
  ]
```

Reply:
I just learned that you can use globs with workspace commands! `npm test --workspace=*` will run on all packages. That might be better than introducing `pnpm`.